specification/deviceregistry/DeviceRegistry.Management/properties/ns_assetProperties.tsp

```diff
@@ -228,6 +266,10 @@ model ManagementGroup {
   @doc("Name of the management group.")
   name: string;
 
+  @doc("Reference to a data source for a given management group.")
```

Comment:
Nit:  better "Reference to a typed or untyped data source which provides context for actions inside the management group"